adagrams/game.py

```diff
@@ -1,11 +1,91 @@
+import random
+import copy
+from collections import Counter
+
+LETTER_POOL = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+}
+
+
+score_table = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+}
+
+
 def draw_letters():
-    pass
+    random_list = []
+    for letter, frequency in LETTER_POOL.items():
+        random_list += list(letter * frequency)
+    return random.sample(random_list, 10)
```

Comment:
Sounds good! Remember that `pop` ends up shifting over everything from where we pop, to the end of the list. To avoid this, since we don't care about the letter order in the merged list, we could swap the letter we want to pop with whatever's currently at the end of the list, then pop at the end which is a constant time operation!